test/CppTests/Tests/BreakpointTests.cs

```diff
@@ -157,7 +157,8 @@ public void FunctionBreakpointsBasic(ITestSettings settings)
         [Theory]
         [DependsOnTest(nameof(CompileKitchenSinkForBreakpointTests))]
         [RequiresTestSettings]
-        [UnsupportedDebugger(SupportedDebugger.Gdb_Cygwin | SupportedDebugger.Gdb_MinGW, SupportedArchitecture.x64 | SupportedArchitecture.x86)]
+        // TODO: Re-enable for Gdb_Gnu
```

Comment:
>// TODO: Re-enable for Gdb_Gnu [](http://example.com/codeflow?start=8&length=30)

I wonder if we need a new attribute for CIDisabled?

Reply:
Actually, do we run these tests anywhere else? If not, probably not worth it.

Reply:
Not yet. There is a small chance someone might be running the tests on their local box where they can have these tests succeeding, but for now I rather take the 57 tests that do run for now :).